hail/python/hailtop/pipeline/task.py

```diff
@@ -347,7 +347,7 @@ def memory(self, memory):
         :class:`.Task`
             Same task object with memory requirements set.
         """
-        self._memory = memory
+        self._memory = str(memory)
```

Comment:
Thanks for making these changes. I had them in a PR that must have gotten closed...

Reply:
Yeah, I also remembered these.  No worries!